arch/arm/core/swap_helper.S

```diff
@@ -140,7 +151,7 @@ SECTION_FUNC(TEXT, __pendsv)
     str r3, [r4]
 #else
     ldr r0, [r2, #_thread_offset_to_basepri]
-    movs.n r3, #0
+    movs r3, #0
```

Comment:
what is the reason for this change?

Reply:
Again, compile error when using the SDK for Cortex-R
src/zephyr/arch/arm/core/swap_helper.S: Assembler messages:
src/zephyr/arch/arm/core/swap_helper.S:158: Error: unexpected character `n' in type specifier
src/zephyr/arch/arm/core/swap_helper.S:158: Error: bad instruction `movs.n r3,#0'

Reply:
@galak ^^